src/components/greetingSlice.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-param-reassign */
+// Import createSlice() from Redux toolkit:
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from '../axios';
+
+// Initial state for Redux store:
+const initialState = {
+  greetings: [],
+  isLoading: true,
+  status: 'idle',
+  error: '',
+};
+
+export const fetchGreetings = createAsyncThunk(
+  'greetings/fetchGreetings',
+  async () => {
+    const response = await axios.get('http://127.0.0.1:3000/api/messages/random_greeting');
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229122251-e491fb8b-23c3-4c9c-8786-c9bbd8038d22.png)

- [ ] I am getting this error from your front end :fearful: Kindly fix it :+1: I have given fixes on the backend pull request :blush: Kindly see them to make your app work :rocket: